src/app/globals.css

```
@@ -2,53 +2,233 @@
+
+/* ── LiveDot ── */
+@keyframes ping {
+  75%, 100% { transform: scale(2); opacity: 0; }
 }
 
-.heading-gradient {
-  background: linear-gradient(85deg, #007bff 18.75%, #fff 84.9%);
-  background-clip: text;
-  -webkit-background-clip: text;
-  -webkit-text-fill-color: transparent;
+@keyframes tooltipFade {
```

Stylelint (17.12.0)

[error] Expected keyframe name "tooltipFade" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)